IBAnimatable/Animatable.swift

```diff
@@ -22,12 +22,12 @@ public protocol Animatable: class {
   /**
    Animation duration (in seconds)
    */
-  var duration: Double { get set }
+  var duration: TimeInterval { get set }
```

Comment:
Thanks Interface Builder.  I hope they can fix this in next version of Xcode to support `TimeInterval`